src/app/services/ws.service.ts

```diff
@@ -350,4 +354,19 @@ export class WebSocketService {
       this.window.location.reload();
     });
   }
+
+  private getLoggedInUserInformation(): void {
+    let authMeUser: DsUncachedUser;
+
+    this.call('auth.me').pipe(
+      switchMap((loggedInUser: DsUncachedUser) => {
+        authMeUser = loggedInUser;
+        return this.call('user.query', [[['uid', '=', authMeUser.pw_uid]]]);
```

Comment:
If this is not a local user, you'll get an empty array in result and will end up spreading undefined.

Reply:
included this case